composer.json

```diff
@@ -8,41 +8,42 @@
     "require": {
         "php": ">=8.0",
         "cweagans/composer-patches": "^1.7",
-        "drupal/core": "^9.4",
-        "drupal/daterange_compact": "^2.0",
-        "drupal/twig_field_value": "^2.0",
+        "drupal/core": "^9.4 || ^10",
+        "drupal/daterange_compact": "^2.0.1",
+        "drupal/twig_field_value": "^2.0.2",
         "openeuropa/composer-artifacts": "^1.0.0-alpha1",
         "openeuropa/oe_bootstrap_theme": "^1.3.0"
     },
     "require-dev": {
         "composer/installers": "^1.11",
-        "drupal/better_exposed_filters": "^5.0",
-        "drupal/composite_reference": "^2.1",
-        "drupal/config_devel": "~1.2",
-        "drupal/core-composer-scaffold": "^9.4",
-        "drupal/core-dev": "^9.4",
+        "drupal/better_exposed_filters": "^6.0.3",
+        "drupal/composite_reference": "^2.2",
+        "drupal/config_devel": "~1.9",
+        "drupal/core-composer-scaffold": "^9.4 || ^10",
+        "drupal/core-dev": "^9.4 || ^10",
         "drupal/description_list_field": "^1.0@alpha",
-        "drupal/entity_reference_revisions": "^1.9",
-        "drupal/extra_field": "^2.1",
-        "drupal/facets_form": "1.0.0-alpha3",
-        "drupal/field_group": "^3.2",
-        "drupal/search_api": "^1.21",
-        "drupal/search_api_autocomplete": "^1.5",
+        "drupal/entity_reference_revisions": "^1.10",
+        "drupal/extra_field": "^2.3",
+        "drupal/facets_form": "1.0.0-alpha6",
```

Comment:
Why this package is still locked in composer.json.

Reply:
Fixed

Reply:
Yes I also don't find anything in the _readme.